### ISSUE TITLE
N00b moment: adding multiple acc's for beehive?

### ISSUE THREAD

**Issue by mvfsullivan** [AHAAAAAAA/PokemonGo-Map#3065]
I have no idea what im doing guys.. But I dont want to give up!!!

I got as far as generating beehive.sh via "python location_generator.py -st 5 -lp 4 -lat 43.360171 -lon -80.313686"

But how do I add multiple accounts, where do I put the info, and what arg' do I use to recall it after??

The instructions I'm reading are confusing the crap out of me -_-

**Comment by Unvicible** [comment]
Did you use the beehive builder.sh?

**Comment by Epion003** [comment]
#2695 Not sure if will help but this is how i set mine up I had some hard time also for awhile since im not a programmer. First you have to have your list of accounts that you are going to use. They should be in a file as      username,password and u save the file as .csv
now u go into the folder that has the beehive location generator and open a cmd window
you will want to put a line similar to this
The example for this for windows is:
python location_generator.py -lat -33.9 -lon 151.1 -st 5 -lp 3 --accounts C:\Documents\accounts.csv --windows --installdir="C:\Downloads\PokemonGo-Map"

accounts is location were you saved users,password 
install is location of the runsever file 
this should be a start once it runs you should have a beehive.bat file in you pokemap folder

**Comment by Silvikk** [comment]
@Epion003 Thanks for your guide! Could you tell me what is the "-lat -33.9 -lon 151.1" ?

**Comment by jtmantis** [comment]
I'm having trouble following that guide as well. Thanks @Epion003 for helping out. I have beehive.bat generated and running. Althouth at the top of it, it says: ERROR: The process "python.exe" not found.

And then

```
E:\Map\PokemonGo-Map-develop>Start "Server" /d E:\Map\PokemonGo-Map-develop /MIN C:\Python27\Python.exe E:\Map\PokemonGo-Map-develop\runserver.py -os -l "54.8985, 23.9036"

E:\Map\PokemonGo-Map-develop>ping 127.0.0.1 -n 6  1>nul
```
And keeps going... Does your work the the same? 

What do you do next? Hod do I make it work on runserver?

**Comment by Epion003** [comment]
@jtmantis yea there first line supposed to kill all python processes but i get that error also still tryin to figure out. but after the comand window will do that line for each of your workers you should see some other windows opening up at the bottom of your screen little python cmd windows...
have you tried opening a web browser and putting in the url: localhost:5000 
to see if it opens a map for u

**Comment by jtmantis** [comment]
Yeah I've tried it to open in a web browser, but it wasn't workinig. Only later I releazed that I didn't put any auth and g map api key in config.ini, lol.

Now it seems to be working fine, althouth some areas look really empty, I wonder if it's just not as accurate as a regular method or it need more workers or something?

**Comment by Epion003** [comment]
@jtmantis haga glad could help with little i knew i ser my turn to ask for help in another thread...
